functions/private/Invoke-WinUtilSSHServer.ps1

```diff
@@ -0,0 +1,80 @@
+function Invoke-WinUtilSSHServer {
+    <#
+    .SYNOPSIS
+        Enables OpenSSH server to remote into your windows device
+    #>
+
+    # Get the latest version of OpenSSH Server
+    $FeatureName = Get-WindowsCapability -Online | Where-Object { $_.Name -like "OpenSSH.Server*" }
+
+    # Install the OpenSSH Server feature if not already installed
+    if ($FeatureName.State -ne "Installed") {
+        Write-Host "Enabling OpenSSH Server"
+        Add-WindowsCapability -Online -Name $FeatureName.Name
+    }
+
+    # Sets up the OpenSSH Server service
+    Write-Host "Starting the services"
+    Start-Service -Name sshd
+    Set-Service -Name sshd -StartupType Automatic
+
+    # Sets up the ssh-agent service
+    Start-Service 'ssh-agent'
+    Set-Service -Name 'ssh-agent' -StartupType 'Automatic'
+
+    # Confirm the required services are running
+    $service = Get-Service -Name sshd
+    $service1 = Get-Service -Name 'ssh-agent'
```

Comment:
Good work! 👍 May i just suggest changing variable names `service` and  `service1` to more meaningful names like `sshDaemonService` and `sshAgentService` or something similar to better indicate what they represent